sdcm/cluster.py

```diff
@@ -1017,14 +1017,14 @@ def config_setup(self, seed_address=None, cluster_name=None, enable_exp=True, en
             scylla_yaml_contents = p.sub('endpoint_snitch: "{0}"'.format(endpoint_snitch),
                                          scylla_yaml_contents)
 
-        if self.is_addition:
+        if self.is_addition is True:
```

Comment:
`is` operator is used to check if two variables point to the same object (Read more here: https://stackoverflow.com/questions/13650293/understanding-pythons-is-operator)
This is not Pythonic way to check if smth is True.